Lib/test/test_fileinput.py

```diff
@@ -83,24 +83,20 @@ def close(self):
 class BufferSizesTests(BaseTests, unittest.TestCase):
     def test_buffer_sizes(self):
         # First, run the tests with default and teeny buffer size.
-        for round, bs in (0, 0), (1, 30):
+        for round in (0, 1):
```

Comment:
No need to repeat the tests twice. Remove also the above comment.

Reply:
Ok.